### PR TITLE
add hover on cart button

### PR DESCRIPTION
**Pull request by OlhaKoziuk**
https://www.loom.com/share/ab41dbf2c8ca467b8617d4c243a672c5?sid=d326f980-4646-4b9b-914c-e91a397fab9c

### CODE REVIEW

**Review by vlevik — APPROVED** [review]
LGTM 👍

**Review by mpohorenyi — APPROVED** [review]
Nice

File: src/modules/CartPage/CartPage.module.scss [hunk 1]
```diff
@@ -107,12 +107,22 @@
...
+      box-shadow: 0px 3px 13px 0px rgba(23, 32, 49, 0.40);
+    }
 
   &__DARK {
     border-radius: 0;
     color: $color__dark-theme__white;
     background-color: $color__dark-theme__accent;
   }
+
+  &__DARK:hover {
```
Comment (dsfreedom) [inline src/modules/CartPage/CartPage.module.scss]: you can just  write inside &__DARK &:hover and it will work nicely

**Review by dsfreedom — APPROVED** [review]
Nice code.)

File: src/modules/CartPage/CartPage.module.scss (hunk 1 above)
Comment (OlhaKoziuk) [reply src/modules/CartPage/CartPage.module.scss]: I'm not sure what you mean. Could you please provide an example for clarification?

Comment (mpohorenyi) [reply src/modules/CartPage/CartPage.module.scss]:
```suggestion
  &__DARK {
    border-radius: 0;
    color: $color__dark-theme__white;
    background-color: $color__dark-theme__accent;
    
    &:hover {
    
    }
  }
```

Comment (OlhaKoziuk) [reply src/modules/CartPage/CartPage.module.scss]: One moment)

**Pull request merged**